lib/pinga-server/src/server.rs

```diff
@@ -419,6 +438,8 @@ async fn execute_job_task(
     metadata: Arc<ServerMetadata>,
```

Comment:
it's the lines write above this, but if we add telemetry to this func, and record the fields you added, we'd be able to do some fancy graphing in honeycomb to show how these numbers change over time!

Reply:
Sorry, not sure I understand

Reply:
oh sorry - this IS in the instrument tag. ignore me!